search-apis/data/results/paracetamol.json

```diff
@@ -0,0 +1 @@
l","fullUrlWithDefaultRoot":"","m":null,"om":null,"pa":3,"pathTerm":null,"pbu":null,"ps":null,"pt":null,"pwbu":null,"q":"paracetamol","rf":null,"s":null,"sa":null,"aggsOnly":null,"sm":null,"sn":null,"sp":null,"spuid":null,"sr":null,"ss":null,"syn":null,"t":null,"td":null,"title":null,"to":null,"route":"search"},"isCurrent":false},{"title":"4","url":{"af":null,"am":null,"arms":null,"ci":null,"cid":null,"co":null,"em":null,"fc":null,"fd":null,"from":null,"fs":null,"fullUrl":"search?pa=4&q=paracetamol","fullUrlWithDefaultRoot":"","m":null,"om":null,"pa":4,"pathTerm":null,"pbu":null,"ps":null,"pt":null,"pwbu":null,"q":"paracetamol","rf":null,"s":null,"sa":null,"aggsOnly":null,"sm":null,"sn":null,"sp":null,"spuid":null,"sr":null,"ss":null,"syn":null,"t":null,"td":null,"title":null,"to":null,"route":"search"},"isCurrent":false},{"title":"5","url":{"af":null,"am":null,"arms":null,"ci":null,"cid":null,"co":null,"em":null,"fc":null,"fd":null,"from":null,"fs":null,"fullUrl":"search?pa=5&q=paracetamol","fullUrlWithDefaultRoot":"","m":null,"om":null,"pa":5,"pathTerm":null,"pbu":null,"ps":null,"pt":null,"pwbu":null,"q":"paracetamol","rf":null,"s":null,"sa":null,"aggsOnly":null,"sm":null,"sn":null,"sp":null,"spuid":null,"sr":null,"ss":null,"syn":null,"t":null,"td":null,"title":null,"to":null,"route":"search"},"isCurrent":false},{"title":"6","url":{"af":null,"am":null,"arms":null,"ci":null,"cid":null,"co":null,"em":null,"fc":null,"fd":null,"from":null,"fs":null,"fullUrl":"search?pa=6&q=paracetamol","fullUrlWithDefaultRoot":"","m":null,"om":null,"pa":6,"pathTerm":null,"pbu":null,"ps":null,"pt":null,"pwbu":null,"q":"paracetamol","rf":null,"s":null,"sa":null,"aggsOnly":null,"sm":null,"sn":null,"sp":null,"spuid":null,"sr":null,"ss":null,"syn":null,"t":null,"td":null,"title":null,"to":null,"route":"search"},"isCurrent":false},{"title":"7","url":{"af":null,"am":null,"arms":null,"ci":null,"cid":null,"co":null,"em":null,"fc":null,"fd":null,"from":null,"fs":null,"fullUrl":"search?pa=7&q=paracetamol","fullUrlWithDefaultRoot":"","m":null,"om":null,"pa":7,"pathTerm":null,"pbu":null,"ps":null,"pt":null,"pwbu":null,"q":"paracetamol","rf":null,"s":null,"sa":null,"aggsOnly":null,"sm":null,"sn":null,"sp":null,"spuid":null,"sr":null,"ss":null,"syn":null,"t":null,"td":null,"title":null,"to":null,"route":"search"},"isCurrent":false}],"next":{"af":null,"am":null,"arms":null,"ci":null,"cid":null,"co":null,"em":null,"fc":null,"fd":null,"from":null,"fs":null,"fullUrl":"search?pa=2&q=paracetamol","fullUrlWithDefaultRoot":"","m":null,"om":null,"pa":2,"pathTerm":null,"pbu":null,"ps":null,"pt":null,"pwbu":null,"q":"paracetamol","rf":null,"s":null,"sa":null,"aggsOnly":null,"sm":null,"sn":null,"sp":null,"spuid":null,"sr":null,"ss":null,"syn":null,"t":null,"td":null,"title":null,"to":null,"route":"search"}},"pageSize":10,"firstResult":1,"lastResult":10,"finalSearchText":"paracetamol","finalSearchTextNoStopWords":"paracetamol","originalSearch":null,"navigators":[],"unfilteredResultsUrl":null}
```

Comment:
Can we separate the search and typeahead responses into different sub folders per service please? There are subtly different fields etc that come back based on the service, for example: cks uses different `TypeAheadType` values for topics etc, and the guidance index has extra fields for different stages of the guidance development process. That way we could potentially use the same 'paracetamol' or 'cancer' search and have different results per service